src/nvidia_rag/rag_server/main.py

```
@@ -303,6 +303,10 @@ def generate(
         reranker_endpoint: str | None = None,
         vlm_model: str | None = None,
         vlm_endpoint: str | None = None,
+        vlm_temperature: float | None = None,
+        vlm_top_p: float | None = None,
+        vlm_max_tokens: int | None = None,
+        vlm_max_total_images: int | None = None,
         filter_expr: str | list[dict[str, Any]] = "",
         enable_query_decomposition: bool | None = None,
```

Ruff (0.14.5)

Boolean-typed positional argument in function definition

(FBT001)